nvdaHelper/localWin10/oneCoreSpeech.cpp

```diff
@@ -30,6 +37,130 @@ using namespace winrt::Windows::Media;
 using namespace winrt::Windows::Foundation::Collections;
 using winrt::Windows::Foundation::Metadata::ApiInformation;
 
+std::recursive_mutex InstanceManager::_instanceStateMutex;

```

Comment:
It seems like it is only `waitUntilReadyForInitialization` / `initializeNewInstance` that requires this to be recursive?